components/molecule/avatar/src/index.js

```diff
@@ -11,68 +17,67 @@ import AvatarBadge, {
   AVATAR_BADGE_SIZES
 } from './AvatarBadge'
 
-export const AVATAR_SIZES = {
-  XXLARGE: 'xxlarge',
-  XLARGE: 'xlarge',
-  LARGE: 'large',
-  MEDIUM: 'medium',
-  SMALL: 'small',
-  XSMALL: 'xsmall'
-}
+import {AVATAR_SIZES} from './settings'
 
-const MoleculeAvatar = ({
-  className: classNameProp,
-  size = AVATAR_SIZES.MEDIUM,
-  skeleton = <AtomSkeleton width="100%" height="100%" animation="wave" />,
-  name,
-  src,
-  fallbackIcon,
-  style,
-  isLoading,
-  children: childrenProp,
-  ...others
-}) => {
-  const baseClassName = 'sui-MoleculeAvatar'
-  const className = cx(baseClassName, `${baseClassName}--${size}`)
-  const backgroundColor = useConvertStringToHex(name)
-  const children = Children.toArray(childrenProp)
-    .filter(child => isValidElement(child))
-    .map(child => cloneElement(child, {size}))
+const MoleculeAvatar = forwardRef(
+  (
+    {
+      className: classNameProp,
+      size = AVATAR_SIZES.MEDIUM,
+      skeleton = <AtomSkeleton width="100%" height="100%" animation="wave" />,
+      name,
+      src,
+      fallbackIcon,
+      style,
+      isLoading,
+      children: childrenProp,
+      ...others
+    },
+    forwardedRef
+  ) => {
+    const baseClassName = 'sui-MoleculeAvatar'
```

Comment:
This can be moved now to `settings.js` file.

Reply:
👍

Reply:
done